packages/web3-eth-accounts/src/account.ts

```diff
@@ -419,11 +632,42 @@ export const create = (): Web3Account => {
 };
 
 /**
- *  Decrypts a v3 keystore JSON, and creates the account.
+ * Decrypts a v3 keystore JSON, and creates the account.
+ * @param keystore - the encrypted Keystore object or string to decrypt
+ * @param password - The password that was used for encryption
+ * @param nonStrict - if true and given a json string, the keystore will be parsed as lowercase.
```

Comment:
```suggestion
 * @param nonStrict - (optional) if true and given a json string, the keystore will be parsed as lowercase.
```

Reply:
same as picture from above, optional is already included in docs so we dont need to write it